edb/server/protocol/ai_ext.py

```diff
@@ -1591,18 +1909,19 @@ async def _handle_rag_request(
             if custom_prompt:
                 if not isinstance(custom_prompt, list):
                     raise TypeError(
-                        "prompt.custom must be a list of {role, content} "
+                        "prompt.custom must be a list"
```

Comment:
```suggestion
                        "prompt.custom must be a list of "
```

Reply:
hm these are possible values, not sure how to add all this into an err message
<img width="384" alt="Screenshot 2024-11-12 at 16 34 05" src="https://github.com/user-attachments/assets/cdf828c9-114e-4b1a-923c-5ee1cd1f994a">